### PR TITLE
1379, 1389, & 1390 kml geolocation track support

### PR DESCRIPTION
**Pull request by kellykelly3**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/geolocation/datamodel/Path.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+import java.util.List;
+import org.sleuthkit.datamodel.BlackboardArtifact;
+import org.sleuthkit.datamodel.Content;
+import org.sleuthkit.datamodel.SleuthkitCase;
+import org.sleuthkit.datamodel.TskCoreException;
+
+/**
+ * Class representing a series of waypoints that form a path.
+ */
+public class Path {
```
[the remark this answers is not in the document]
Comment (kellykelly3) [reply Core/src/org/sleuthkit/autopsy/geolocation/datamodel/Path.java]: I know.  I was a little stumped on a name for that class.  I was thinking like a path that you following.  I am open to suggestions.

File: Core/src/org/sleuthkit/autopsy/modules/drones/DATExtractor.java [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+                String[] values = line.split(","); //NON-NLS
+                GeoTrackPoint point = createTrackPoint(headerMap, values);
+                if (point != null) {
+                    trackPoints.add(point);
+                }
+            }
+
+        } catch (IOException ex) {
+            throw new DroneIngestException(String.format("Failed to read DAT csvFile %s", csvFilePath), ex); //NON-NLS
```
Comment (raman-bt) [inline Core/src/org/sleuthkit/autopsy/modules/drones/DATExtractor.java]: May be log the objectID of the file in addition to path/name.

File: Core/src/org/sleuthkit/autopsy/modules/drones/DroneExtractor.java [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+     */
+    protected File getTemporaryFile(IngestJobContext context, AbstractFile file) throws DroneIngestException {
+        String tempFileName = file.getName() + file.getId() + file.getNameExtension();
+
+        Path tempFilePath = Paths.get(getExtractorTempPath().toString(), tempFileName);
+
+        try {
+            ContentUtils.writeToFile(file, tempFilePath.toFile(), context::dataSourceIngestIsCancelled);
+        } catch (IOException ex) {
+            throw new DroneIngestException(String.format("Unable to create temp file %s for abstract file %s", tempFilePath.toString(), file.getName()), ex); //NON-NLS
```
Comment (raman-bt) [inline Core/src/org/sleuthkit/autopsy/modules/drones/DroneExtractor.java]: log the object id as well.

File: Core/src/org/sleuthkit/autopsy/geolocation/datamodel/Path.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (rcordovano) [reply Core/src/org/sleuthkit/autopsy/geolocation/datamodel/Path.java]: How about GeoPath?

**Pull request merged**